src/EnergyPlus/WaterCoils.hh

```diff
@@ -117,6 +117,7 @@ namespace WaterCoils {
     extern Array1D_int PartWetCoolCoilErrs;   // error counting for detailed coils
     extern bool GetWaterCoilsInputFlag;       // Flag set to make sure you get input once
     extern Array1D_bool CheckEquipName;
+    // extern bool WaterCoilControllerCheckOneTimeFlag; // flg used to check water coil controller
```

Comment:
This should just be removed if it's not used anymore unless there is a good reason to keep it.